source/_integrations/esphome.markdown

```diff
@@ -42,6 +42,9 @@ works_with:
 
 This integration allows [ESPHome](https://esphome.io) devices to connect directly to Home Assistant with the [native ESPHome API](https://esphome.io/components/api.html).
 
+
+`Note: This system uses MDNS for ease of use on Home networks. Multiple subnets and vlans will require the use of additional options for devices to show online. Please see the forums for needed additions to configurations to allow devices to be detected.`
```

Comment:
what do you mean by *show online*? Are you referring to the status shown in the addon here
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/0b0d5403-46a1-4c4d-80d5-ab6b70502679)

Reply:
yes that is the screen mentioned.  The problem is esphome treats that as gospel and does not pole or update the device unless you know exactly what to change in the config. if you have no idea that its due to a combination of configuration/development choices and the fact that you are not any joe off the street and have more than expcted by the developer it makes it hard for NEW users to Home Assistant to know what to look for to fix their issue.   To me esphome was some crappy slaped together project due to my first impression of  [tasmota installs and works with minimal configuration, esphome does not on the same device.]